### ISSUE TITLE
record struct parameter list may not be empty?

### ISSUE THREAD

**Issue by tmds** [dotnet/docs#27884]
This page says:
```
At most one partial type declaration of a partial record struct may provide a parameter_list. The parameter_list may not be empty.
```

Is the last sentence correct? This compiles fine:

```cs
partial record struct Person()
{ }

partial record struct Person
{ }
```

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ff68d937-96a7-1cec-28e9-4f47e34d1bdf
* Version Independent ID: e1103d21-a9f8-787f-3c14-0d0581f92675
* Content: [Record structs - C# 10.0 draft specifications](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-10.0/record-structs#feedback)
* Content Source: [proposals/csharp-10.0/record-structs.md](https://github.com/dotnet/csharplang/blob/main/proposals/csharp-10.0/record-structs.md)
* Product: **dotnet-csharp**
* Technology: **csharp-spec**
* GitHub Login: @dotnet-bot
* Microsoft Alias: **wiwagn**

**Comment by Youssef1313** [comment]
Probably it's outdated. If you'd like to contribute, you can open a PR on dotnet/csharplang to get this fixed. Here is the file:

https://github.com/dotnet/csharplang/blob/main/proposals/csharp-10.0/record-structs.md

**Comment by BillWagner** [comment]
adding @cston 

I looked through the LDM notes, and I didn't see any decision addressing this. What is the correct fix?

**Comment by cston** [comment]
Both examples are supported. If the parameter list is empty, a parameterless primary constructor is generated; if the parameter list is missing, no primary constructor is generated.

I've created a PR to update the documentation: https://github.com/dotnet/csharplang/pull/5670

cc @jcouv

**Comment by BillWagner** [comment]
Thanks @cston 

Fixed in dotnet/csharplang#5670
